GUIDE_CONFIG.md

```diff
@@ -116,27 +133,40 @@ export default ({ cwd, development }) => {
   db: {
     host: {
       doc: "Database host name/IP",
-      format: "*",
-      default: "server1.dev.test",
-    },
+        format
+    :
+      "*",
```

Comment:
this formatting is super weird

Reply:
yeah.... my editor did that 😆 I'll try to amend it

Reply:
`vim` ❤️